src/lib/common/sol-platform-impl-contiki.c

```diff
@@ -135,3 +135,31 @@ sol_platform_impl_umount(const char *mpoint, void (*cb)(void *data, const char *
     SOL_WRN("Not implemented");
     return -ENOTSUP;
 }
+
+int
+sol_plataform_unregister_hostname_monitor(void)
+{
+    SOL_WRN("Not implemented");
+    return -ENOTSUP;
```

Comment:
we're not sure the OS have such concept. If they do not, then what we can do is a "fallback" implementation that only stores/reads from a global variable, so other soletta apps/subsys would work